index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
+            <nav class="toolbar">
+                <a id="hamburger" href="#"><img src="icons/menu.svg"></a>   
+            </nav>
+        </header>
+        <main>
+            <article class="headline-section">
+                    <h2>Hey There.<br>I’m Alex</h2>
+                    <h4>I’am a Software Developer</h4>
```

Comment:
- [ ] The main title("Hey There. I’m Alex") should be an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly change that.